core/domain/email_manager.py

```diff
@@ -1159,6 +1242,113 @@ def send_mail_to_notify_users_to_review(user_id, category):
             email_subject, email_body, feconf.NOREPLY_EMAIL_ADDRESS)
 
 
+def send_mail_to_notify_contributor_dashboard_reviewers(
+        reviewer_ids, reviewers_suggestion_email_infos):
+    """Sends an email to each Contributor Dashboard reviewer notifying them of
+    the suggestions that have been waiting the longest for reivew, that the
+    reviewer has permission to review.
+
+    Args:
+        reviewer_ids: list(str). A list of the Contributor Dashboard reviewer
+            user ids to notify.
+        reviewers_suggestion_email_infos:
+            list(list(ReviewableSuggestionEmailInfo)). A list of suggestion
+            email content info objects for each reviewer. These suggestion
+            email content info objects contain the key imformation about the
```

Comment:
```suggestion
            email content info objects contain the key information about the
```

I suggest proof-reading all your docstrings for these types of errors.

Reply:
Yeah sorry - normally I do, my bad!